openlibrary/plugins/upstream/addbook.py

```diff
@@ -629,7 +629,9 @@ def GET(self, key):
                 'key': '',
                 'type': {'key': '/type/work'},
                 'title': edition.title,
-                'authors': [{'type': '/type/author_role', 'author': {'key': a['key']}} for a in edition.get('authors', [])]
+                'authors': [{'type': '/type/author_role', 'author': {'key': a['key']}} for a in edition.get('authors', [])],
+                'subjects': edition.get('subjects', []),
```

Comment:
this lgtm -- though is it possible for `covers` to be `cover_url`?

Reply:
I meant to take the list of integer cover ids, which is the standard way the UI gets and displays covers. I don't think `cover_url` is a standard field on stored works or editions.

Reply:
ok, this makes sense, lgtm